src/js/index.js

```diff
@@ -34,24 +72,67 @@ const makeMenuTemplate = menuName => {
   $espressoMenuList.appendChild($li);
 };
 
+// 메뉴 리스트 렌더링
+const renderMenu = () => {
+  if (!localStorage.getItem('menu')) return;
+
+  const selectedMenuList = JSON.parse(localStorage.getItem('menu'))[
+    selectedMenu
+  ];
+
+  $espressoMenuList.innerHTML = '';
+  selectedMenuList.map(({ name, soldOut }) => makeMenuTemplate(name, soldOut));
+  updateTotalMenuNum();
+};
+
 // 메뉴 추가
 const addNewMenu = () => {
   const $input = document.getElementById('espresso-menu-name');
   const menuName = $input.value;
+  let selectedMenuList = [];
 
   if ($input.value.trim() === '') {
     $input.value = '';
     return;
   }
 
-  makeMenuTemplate(menuName);
+  // localStorage
+  if (localStorage.getItem('menu')) {
+    menuObj = JSON.parse(localStorage.getItem('menu'));
+    selectedMenuList = menuObj[selectedMenu];
+  }
+  selectedMenuList.push({
+    name: menuName,
+    soldOut: false,
+  });
+  menuObj[selectedMenu] = selectedMenuList;
+  localStorage.setItem('menu', JSON.stringify(menuObj));
+
+  renderMenu();
   $input.value = '';
-  updateTotalMenuNum();
+};
+
+// 메뉴 변경에 따른 localStorage 업데이트
+const updateLocalStorageMenu = updatedMenuList => {
+  menuObj = JSON.parse(localStorage.getItem('menu'));
+  menuObj[selectedMenu] = updatedMenuList;
+  localStorage.setItem('menu', JSON.stringify(menuObj));
+};
+
+// 메뉴 품절 여부 토글
+const toggleSoldOut = ($li, idx, selectedMenuList) => {
+  const $menuName = $li.firstElementChild;
+  $menuName.classList.toggle('sold-out');
+
+  // localStorage
+  const isSoldOut = selectedMenuList[idx]['soldOut'];
+  selectedMenuList[idx]['soldOut'] = isSoldOut ? false : true;

```

Comment:
```suggestion
  selectedMenuList[idx]['soldOut'] = !isSoldOut;
```
이렇게 boolean으로 간단하게 표현할 수 있을것 같아요~!